testsuite/features/support/setup_browser.rb

```diff
@@ -88,6 +88,11 @@ def enable_assertions
   scenario.skip_invoke! unless $sle15_minion
 end
 
+# do we have scc credentials
+Before('@SCC_CREDS') do |scenario|
+  scenario.skip_invoke! unless ENV['SCC_CREDS']
```

Comment:
`ENV['SCC_CREDENTIALS']`

This might be slow to lookup the `ENV[]` array at every scenario. For the other tags, we go through a global ruby variable, which could be `$scc_credentials` in your case. Such variables are set up in `support/twopence_init.rb`. It is what I recommended to do in the card. What about doing it like this?